invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/components/ResultsList/ResultsList.js

```diff
@@ -15,33 +15,86 @@ export class ResultsList extends Component {
     this.fetchPatronCurrentLoans = this.props.fetchPatronCurrentLoans;
   }
 
+  state = { open: false };
+
+  open_modal = () => () => {
```

Comment:
Rename this to `openModal`? I also don't think it's necessary to have an arrow function inside an arrow function here since you can just pass the function object as the `onClick` callback (like in `close_modal`).